MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -525,6 +525,20 @@ public string DisplayTitle
         /// <value>The real frame rate.</value>
         public float? RealFrameRate { get; set; }
 
+        /// <summary>
+        /// Gets the framerate used as reference.
+        /// Prefer AverageFrameRate, if that is null or an unrealistic value
+        /// then fallback to RealFrameRate.
+        /// </summary>
+        /// <value>The reference frame rate.</value>
+        public float? ReferenceFrameRate
+        {
+            get
+            {
+                return AverageFrameRate < 1000 ? AverageFrameRate : RealFrameRate;
```

Comment:
I have not found a ticket that accurately reflects this issue on ffmpeg tracker. The one that most accurately reflects this is the bug report on OBS forum: 

https://obsproject.com/forum/threads/mkv-files-are-saving-with-a-1000-frames-second.64311


I think this is not an ffmpeg only issue because I've seem players with mpv backend also struggles to get the correct fps:

<img width="209" alt="image" src="https://github.com/user-attachments/assets/da5d79f4-c721-46b8-a04b-bb62967cfb32">

Perhaps this is just a quirk of the matroska reading/writing and some libs are just not compatible with each other.

Reply:
Then maybe leave some comments about this quirk ourselves.

Reply:
Shouldn't this workaround only be applied if the container is MKV?

Reply:
It is quite impossible to do that because the stream is decoupled from the original container, and I don't see any negative impact to apply this globally. Do you have specific use case where you do need this?